kitsune/search/v2/es7_utils.py

```diff
@@ -21,7 +21,15 @@ def _insert_custom_filters(locale, filter_list, char=False):
     def mapping_func(position_filter_tuple):
         position, filter = position_filter_tuple
         if type(filter) is dict:
-            name = f'{locale}_{position}_{filter["type"]}'
+            prefix = locale
```

Comment:
The first argument of this function is accepting a locale. With this optimization you basically adapt a new naming schema by passing the name of the default analyzer as a locale. Personally I believe that the complexity introduced counteracts the benefits. In any case though we should not treat the analyzers name as a locale

Reply:
To be clear, I am not against having a default name where applicable for our filters but how we are doing it should be more straightforward